### DISCUSSION TITLE
Could you please add a way to use the "Reminder" plugin without creating a task?

### DISCUSSION THREAD

**Discussion by ricardonevesmusic** [uphy/obsidian-reminder#163] [Ideas]
Hi.

Could you please add a way to use the "Reminder" plugin without creating a task?

Was it possible to do this in the future.

It would be nice.

Also, could you allow the time of the notification to stay exactly the same, but to be moved to the next day, or any other specified day?

Right now to do this I have to "mute" the reminder and change the date manually.

Yeah, that's it.

Just some food for thought.

It would be nice if it got added to the plugin (if not, no problem either, but it would just be a nice addition).

Thank you.

**Comment by jcamp** [comment]
Definitely second this. Just being able to add a date to non-tasks would be massive. Even if they just show in the calendar and do not notify or have mechanisms to show. If listed in the calendar would be perfect. Perhaps have an icon to indicate that the calendar item is a non-checkbox item (as such will not form part of reminder popups, etc.)

**Comment by texegon** [comment]
I also would love it if you could add an option to add a reminder for entire notes. Preferably by linking to a Date or Date/Time-property.
